doc/source/extras/pyinstaller.rst

```diff
@@ -1,6 +1,6 @@
 .. _ref_pyinstaller:
 
-Freezing PyVista with pyinstaller
+Freezing PyVista with Pyinstaller
```

Comment:
They seem to style themselves as [PyInstaller](https://pyinstaller.org/en/stable/)
```suggestion
Freezing PyVista with PyInstaller
```